infra/base-images/base-builder/detect_repo.py

```diff
@@ -29,25 +29,37 @@
 
 
 def main():
-  """Function to get a git repos information based on its commit."""
+  """Function to get a git repos information based on its commit.
+
+  Raises:
+    ValueError when a commit or a ref is not provided
+  """
   parser = argparse.ArgumentParser(
       description='Finds a specific git repo in an oss-fuzz projects docker file.'
   )
   parser.add_argument(
       '--src_dir',
       help='The location of the oss-fuzz projects source directory',
       required=True)
+  parser.add_argument(
```

Comment:
I'm confused, why do we need repo name at all?

Reply:
It is for the case when the repo name is different than the OSS-Fuzz project name e.g. ndpi vs. nDPI.